tests/database/fixtures.py

```
@@ -33,19 +23,15 @@ def _ping_database(engine: Engine) -> bool:
-    _init_db(engine)
-
-    return engine
+def engine() -> Generator[Engine, None, None]:
+    with PostgresContainer("postgres:18") as pg:
+        connection_string = pg.get_connection_url(driver="psycopg")
+        engine = create_engine(connection_string)
+        _init_db(engine)
+        yield engine
 
 
 @pytest.fixture(scope="function")
```

Ruff (0.15.10)

[warning] `scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)